docs/guides/docker_compose.md

```diff
@@ -6,23 +6,21 @@ nav_order: 2
 last_modified_at: 2023-05-05T10:01:00+02:00
 ---
 
-> :warning: **This page is work in progress!**
+## Docker Compose for FDM Monster
 
-# Running FDM Monster with docker-compose
+This is a guide on how to use Docker Compose to run FDM Monster. Please note that this page is a work in progress.
 
-Docker is a great tool! Using `docker-compose` is better and `portainer` is most awesome! Please read the following before continuing:
-1) NOTE we assume you are familiar with `docker` and `docker-compose`. These are great tools for isolating your software deployment (+), but it be quite new to some users (-).
-    - We cannot support each custom scenario or setup!
-    - Take good care of checking your device's memory limits, architecture and CPU power (`docker stats` and for example `mem_limit: 400m` for 400MB limit in docker-compose)
```

Comment:
The 400MB condition should be removed as the flag is not supported in compose v3+. Same goes for the actualy .yml content below.